### PR TITLE
Update namespace

### LINKED ISSUE #69: Switch namespace
Switch all packages to the new `org.dataflowanalysis` namespace

### PR DESCRIPTION
**Pull request by Entenwilli**
This PR changes the namespace of the analysis and all bundles, tests and features of this Repository.
Old namespace: `org.palladiosimulator.dataflow.confidentiality.analysis`
New namespace: `org.dataflowanalysis.analysis`
This PR closes #69

### CODE REVIEW

File: bundles/org.dataflowanalysis.analysis/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
-Bundle-Name: Data Flow Confidentiality Analysis
-Bundle-SymbolicName: org.palladiosimulator.dataflow.confidentiality.analysis
+Bundle-Name: Data Flow Analysis
+Bundle-SymbolicName: org.dataflowanalysis.analysis
 Bundle-Version: 1.0.0.qualifier
 Automatic-Module-Name: org.palladiosimulator.dataflow.confidentiality.analysis
```
Comment (Nicolas-Boltz) [inline bundles/org.dataflowanalysis.analysis/META-INF/MANIFEST.MF:6]: automatic-module-name also needs to be changed

File: tests/org.dataflowanalysis.analysis.testmodels/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -1,10 +1,10 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
-Bundle-Name: Confidentiality Analysis Test Models
-Bundle-SymbolicName: org.palladiosimulator.dataflow.confidentiality.analysis.testmodels
+Bundle-Name: Data Flow Analaysis Testmodels
+Bundle-SymbolicName: org.dataflowanalysis.analysis.testmodels
 Bundle-Version: 1.0.0.qualifier
 Automatic-Module-Name: org.palladiosimulator.dataflow.confidentiality.analysis.testmodels
```
Comment (Nicolas-Boltz) [inline tests/org.dataflowanalysis.analysis.testmodels/META-INF/MANIFEST.MF:6]: also change automatic-module-name

**Review by Nicolas-Boltz — CHANGES_REQUESTED** [review]
Two small spots missed

File: bundles/org.dataflowanalysis.analysis/META-INF/MANIFEST.MF (hunk 1 above)
Comment (Entenwilli) [reply bundles/org.dataflowanalysis.analysis/META-INF/MANIFEST.MF]: Renamed all three automatic-module-names 👍🏼

File: tests/org.dataflowanalysis.analysis.testmodels/META-INF/MANIFEST.MF (hunk 2 above)
Comment (Entenwilli) [reply tests/org.dataflowanalysis.analysis.testmodels/META-INF/MANIFEST.MF]: Renamed all three automatic-module-names 👍🏼

**Review by Nicolas-Boltz — APPROVED** [review]
Looks good to me

**Pull request merged**